zeek/plugin/include/zeek-compat.h

```diff
@@ -11,8 +11,11 @@
 
 #include <zeek-spicy/autogen/config.h>
 
-// TODO: Move to <zeek/zeek-config.h> once we don't need to support Zeek < 3.2 anymore.
+#if SPICY_ZEEK_VERSION_NUMBER >= 30200 // Zeek >= 3.2
```

Comment:
Why do we need to introduce `SPICY_ZEEK_VERSION_NUMBER` for this while using `ZEEK_VERSION_NUMBER` below? Aren't those the same?

Reply:
`ZEEK_VERSION_NUMBER` isn't available in this `zeek-compat.h` until it includes `zeek-config.h`, and whether to include that via `zeek/zeek-config.h` depends on the Zeek version number.

`SPICY_ZEEK_VERSION_NUMBER` and  `ZEEK_VERSION_NUMBER` *should* be the same, but come from different locations: former comes from Spicy's CMake having run `zeek-config --version` and the later comes from  what `zeek-config.h` actually reports via preprocessor definition.

(Expect this gets simplified when older Zeeks become unsupported).

Reply:
got it now, thanks